src/AzureAppConfigurationImpl.ts

```diff
@@ -657,3 +711,7 @@ function getValidFeatureFlagSelectors(selectors?: SettingSelector[]): SettingSel
         return getValidSelectors(selectors);
     }
 }
+
+function isFailoverableError(error: any): boolean {
+    return isRestError(error) && (error.statusCode === 408 || error.statusCode === 429 || (error.statusCode !== undefined && error.statusCode >= 500));

```

Comment:
https://github.com/Azure/AppConfiguration-DotnetProvider/issues/568

We treat 401 & 403 as failoverable error too.